search/interfaces/graphql/dto/facets.go

```diff
@@ -0,0 +1,330 @@
+package dto
```

Comment:
Didn't the frequent usage of the same package name broke the gqlgen generation? Would therefore rename the package to `searchdto` or something more unique.